src/main/perl/Path.pm

```diff
@@ -346,6 +352,21 @@ sub any_exists
     return $path && (-e $path || -l $path);
 }
 
+=item is_symlink
+
+Test if C<path> is a symlink.
+
+Returns true as long as C<path> is a symlink, even though the 
+symlink target doesn't exist.
+
+=cut
+
+sub is_symlink
```

Comment:
A hardlink is just another directory entry for an inode. There is no way to distinguish the "primary entry" from the new one (or the other ones). So there is no way to say that one particular file (directory entry) is a hardlink. `has_hardlink()` is probably feasable: just count the number of directory entries for an inode and if >1, means that the entry has hardlinks. Let's keep it for when we need it...